src/redux/migrations.ts

```diff
@@ -823,4 +824,11 @@ export const migrations = {
       sendingFiatAccount: false,
     },
   }),
+  79: (state: any) => ({
+    ...state,
+    fiatConnect: {
+      ..._.omit(state.fiatConnect, 'sendingFiatAccount'),
```

Comment:
it seems fine to set `sendingFiatAccountStatus` to `NotSending` when `sendingFiatAccount` is `false`, but what about when it is `true`?

Reply:
seems like a more complex migration might be needed in that case

Reply:
or if it's expected to never happen, since migrations occur at a time when it's not possible to be actively sending fiat account details to the server, maybe log an error if it does occur?

Reply:
> expected to never happen, since migrations occur at a time when it's not possible to be actively sending fiat account details to the server

I think this is correct. I don't think its something that we need an error case to handle. Especially if we codify this as expected behavior by adding sendingFiatAccountStatus to rehydrate.

Reply:
Yeah, there is only a very brief window of time where `sendingFiatAccount` is anything but `false`; it will always be `false` on app start.